apps/docusaurus/docs/guides/keyless-accounts.md

```diff
@@ -59,9 +133,11 @@ At a high level, there are three steps to follow in order to integrate Keyless A
    2. Instantiate the user’s `KeylessAccount`
    3. Sign and submit transactions via the `KeylessAccount`.
 
-## Step 1. Configure your OpenID integration with your IdP
+### Step 1. Configure your OpenID integration with your IdP
+
+#### Supported Identity Providers
```

Comment:
Nit: Use steps component https://preview.aptos.dev/en/developer-platforms/contribute/components/steps

Reply:
will do when port to nextra